targets/targets.json

```diff
@@ -1066,6 +1066,7 @@
         "inherits": ["FAMILY_STM32"],
         "core": "Cortex-M7F",
         "extra_labels_add": ["STM32F7", "STM32F756", "STM32F756xG", "STM32F756ZG"],
+        "macros_add": ["MBEDTLS_CONFIG_HW_SUPPORT", "MBEDTLS_MD5_C", "MBEDTLS_CONFIG_HW_SUPPORT"],
```

Comment:
Hi, This should be : 
`"macros_add": ["USBHOST_OTHER","MBEDTLS_CONFIG_HW_SUPPORT"],`
The MBEDTLS_MD5_C should be added in the mbed_app.json if needed.
Cheers